tests/integration/test_orders_api.py

```diff
@@ -536,6 +536,222 @@ async def test_download_asset_md(tmpdir, session):
+                                     "1975b76cc6904ff84f432be2ef1fea2e",
+                                     "sha256":
+                                     "300ebdfd62614f82ce6ba45b298bf797\
+                                           f1ea595d9214bcf23899f3fd2267c5dd"
+                                 },
+                                 headers={
+                                     'Content-Type':
+                                     'application/json',
+                                     'Content-Disposition':
+                                     'attachment; filename="asset1.json"'
+                                 })
+    respx.get(dl_url1).return_value = mock_asset1
+
+    mock_asset2 = httpx.Response(HTTPStatus.OK,
+                                 json={
+                                     "md5":
+                                     "27323e74b2590f97734124bd4cce055e",
+                                     "sha256":
+                                     "cc70916fe9af731d80220aec6dfbf7c\
+                                          91b9339fea61bf2a6b4d83cee6e11f7c9"
+                                 },
+                                 headers={
+                                     'Content-Type':
+                                     'application/json',
+                                     'Content-Disposition':
+                                     'attachment; filename="asset2.json"'
+                                 })
+    respx.get(dl_url2).return_value = mock_asset2
+
+    mock_manifest = httpx.Response(
+        HTTPStatus.OK,
+        json={
+            "name":
+            "",
+            "files": [{
+                "path": "asset1.json",
+                "digests": {
+                    "md5":
+                    "9a967dfe809c2eb25d990ad547feadf0",
+                    "sha256":
+                    "16aa5141ae5f3e739de78bf1576d4474fc16de471414ce74e793dc852f93d7a8"
+                },
+            },
+                      {
+                          "path": "asset2.json",
+                          "digests": {
+                              "md5":
+                              "8092eb7d64064d4c0c2136d4cdb61eaa",
+                              "sha256":
+                              "0a6874c3e4a75c45828fc3c2ae5f6c238b9f99bfcdcb020332b02e651c5c2873"
+                          }
+                      }]
+        },
+        headers={
+            'Content-Type': 'application/json',
+            'Content-Disposition': 'attachment; filename="mock_manifest.json"'
+        })
+    respx.get(dl_url3).return_value = mock_manifest
+    # Test Checksum
+    for checksum in ['MD5', 'SHA256']:
+        cl = OrdersClient(session, base_url=TEST_URL)
+        filenames = await cl.download_order(oid, directory=str(tmpdir))
+        cl.calculate_checksum(manifest_data=mock_manifest.json(),
+                              filenames=filenames,
+                              checksum=checksum)
```

Comment:
@mkshah605 instead of calling calculate_checksum explicitly, this test should call download_order with the `checksum` keyword argument.